app/renderer/js/main.ts

```diff
@@ -961,6 +985,19 @@ class ServerManagerView {
 		ipcRenderer.on('new-server', () => {
 			this.openSettings('AddServer');
 		});
+
+		ipcRenderer.on('mute-org', (event: Event, index: number) => {
+			if (DomainUtil.getDomain(index).muted) {
+				// server is already muted
+				this.tabs[index].webview.updateBadgeCount(DomainUtil.getDomain(index).badgeCount);
+				DomainUtil.updateBadgeCount(index, -1);
```

Comment:
Can you explain the -1 to me here? As far as I understand, you're saving the `badgeCount` in `domain.json` so that you can instantly show the correct message count when the user decides to unmute an organisation. 
Is there a problem with saving the current `badgeCount` instead of -1?

Reply:
Yaa, the badgecount is updated in domain.json to instantly show the correct message count when the user decides to unmute an organization, updating the value to -1 denotes that the particular organization is not muted. This is maintained to avoid confusion and to keep a check if the value is updated on muting and unmuting an organization.